presto-blackhole/src/test/java/com/facebook/presto/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -187,6 +192,40 @@ public void fieldLength()
         assertThatQueryReturnsValue("DROP TABLE nation", true);
     }
 
+    @Test
+    public void pageProcessingDelay()
+            throws Exception
+    {
+        Session session = testSessionBuilder()
+                .setCatalog("blackhole")
+                .setSchema("default")
+                .build();
+
+        Duration pageProcessingDelay = Duration.valueOf("2s");
```

Comment:
I'm going to decrease the timeout to `1s`. It will have slightly higher chance of false-positive result, but i believe it will be still fine.